packages/gensx/src/commands/run.ts

```diff
@@ -45,10 +53,25 @@ export async function runWorkflow(
       }
     }
 
+    if (!environmentName) {
+      const selectedEnvironment = await getSelectedEnvironment(projectName);
+      if (selectedEnvironment) {
+        environmentName = selectedEnvironment;
+        spinner.info(
+          `Using environment name from CLI: ${pc.cyan(environmentName)}`,
+        );
+      } else {
+        spinner.fail("No environment name provided");
+        throw new Error(
+          "No environment name found. Either specify --environment or run `gensx environment select <environment>`.",
+        );
```

Comment:
Can this fallback to the selection/creation prompt flow?

Reply:
Yeah I can make that update. I was worried that it would be too annoying to see every time you ran something but maybe better safe than sorry.

Reply:
Well these are commands that only interact with our API, I'm not sure why you would run them without logging in...

I agree that we should not prompt/require it for running the dev server.

Reply:
updated this to fallback to the same selection/creation flow. refactored so they could use the same utility function.